### DISCUSSION TITLE
Change background color

### DISCUSSION THREAD

**Discussion by rbruno1** [jspsych/jsPsych#1579] [Q&A] [answered]
I want absolutely no whiteness on the page while taking the experiment. Is there a way that this can be done?

**Comment by becky-gilbert** [comment]
Try setting the background color of the `body` element. One way to do this is by adding `style` tags to your HTML page:
```html
<style>
  body {background-color: black; color: white;}
</style>
```
Here `color: white;` is setting the default font color (otherwise the text won't be visible).

**Comment by rbruno1** [comment, marked as answer]
Do you know how i would do this is cognition.run?

Best,
Ryan

**Reply by jodeleeuw** [reply]
You can create a .css file and upload it here:

![image](https://user-images.githubusercontent.com/595524/109374229-bd7afb80-7881-11eb-90a6-54867c7ba311.png)

The CSS file would just have the one line

```css
body {background-color: black; color: white;}
```

**Reply by rbruno1** [reply]
Thank you. Now I am trying to figure out how to make the experiment full screen. I know i have to make a fullscreen trial but whenever I do that it either doesnt show or the experiment doesnt work.

**Comment by rbruno1** [comment]
Thank you.

Best,
Ryan

On Feb 26, 2021, at 21:28, Josh de Leeuw <notifications@github.com> wrote:

﻿
Rush Email Security

**WARNING** This email originated from outside of Rush University Medical Center. **DO NOT CLICK* links or attachments unless you recognize the sender and know the content is safe. Remember, Rush IS will never ask for user ID information via email communication.

You can create a .css file and upload it here:

[image]<https://urldefense.com/v3/__https://user-images.githubusercontent.com/595524/109374229-bd7afb80-7881-11eb-90a6-54867c7ba311.png__;!!OlavHw!rr1tGu7vaa8frDH8D7CtDFkqiz2RzWf32FG1EGOuYX3lJtpOOMwm56qIycl9HDjXsw$>

The CSS files would just have the one line

body {background-color: black; color: white;}

**Comment by rbruno1** [comment]
How do you create a fullscreen trial in cognition.run?

Best,
Ryan

On Feb 26, 2021, at 21:59, Ryan Bruno <Ryan_Bruno@rush.edu> wrote:

﻿ Thank you.

Best,
Ryan

On Feb 26, 2021, at 21:28, Josh de Leeuw <notifications@github.com> wrote:

﻿
Rush Email Security

**WARNING** This email originated from outside of Rush University Medical Center. **DO NOT CLICK* links or attachments unless you recognize the sender and know the content is safe. Remember, Rush IS will never ask for user ID information via email communication.

You can create a .css file and upload it here:

[image]<https://urldefense.com/v3/__https://user-images.githubusercontent.com/595524/109374229-bd7afb80-7881-11eb-90a6-54867c7ba311.png__;!!OlavHw!rr1tGu7vaa8frDH8D7CtDFkqiz2RzWf32FG1EGOuYX3lJtpOOMwm56qIycl9HDjXsw$>

The CSS files would just have the one line

body {background-color: black; color: white;}
